.github/workflows/superset-python-unittest.yml

```diff
@@ -0,0 +1,141 @@
+# Python MySQL unit tests
```

Comment:
I'm mostly indifferent to having these split up versus all in one file, but I'd like to avoid shuffling these around unnecessarily to avoid losing the git history.

Reply:
There was a hypothesis that there are overheads involved in scheduling multiple workflows vs running them in one. Not sure how much it helps but I still think it makes sense to combine these three considering they are very similar in nature and not particularly expensive to run.